app/controllers/users_controller.rb

```diff
@@ -12,18 +13,11 @@ class UsersController < ApplicationController
   def index
     @user = current_user
     @gds = current_user.permissions.admin_management
-
-    if @gds
```

Comment:
Am happy this nested if is gone, I thought the index was doing too much, so `emails_csv` is really good